packages/reserve-site/src/components/ReserveAddresses.tsx

```diff
@@ -0,0 +1,79 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core'
+import * as React from 'react'
+import Button from 'src/components/Button'
+import CopyIcon from 'src/components/CopyIcon'
+interface Props {
+  dai: string
+  btc: string
+  celo: string
+  custody: string
+  eth: string
+}
+
+export default function ReserveAddresses({ dai, btc, celo, custody, eth }: Props) {
+  return (
+    <>
+      <Address label="CELO" hex={celo} />
+      <Address label="CELO in Custody" hex={custody} />
+      <Address label="BTC" hex={btc} />
+      <Address label="ETH" hex={eth} />
+      <Address label="DAI" hex={dai} />
+      <Button href="https://docs.celo.org/command-line-interface/reserve">
+        Query Reserve Holdings
+      </Button>
+    </>
+  )
+}
+const MILLISECONDS = 5000
+
+function useCopy(hex) {
+  const [justCopied, setCopied] = React.useState(false)
+
+  function onPress() {
```

Comment:
Tiny nit: use arrow func for inline functions

Reply:
curious is there a reason why arrow functions are better here?